plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/complete-onboarding-wizard.spec.js

```diff
@@ -71,56 +78,56 @@ test.describe('Store owner can complete onboarding wizard', () => {
 			storeDetails.us.products
 		);
 
-		await onboarding.unselectBusinessFeatures(page);
-		await page.click('button >> text=Continue');
-	});
+		await onboarding.unselectBusinessFeatures( page );
+		await page.click( 'button >> text=Continue' );
+	} );
 
-	test('can complete the theme selection section', async ({ page }) => {
+	test( 'can complete the theme selection section', async ( { page } ) => {
 		await page.goto(
 			'wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard&step=theme'
 		);
 		const pageHeading = await page.textContent(
 			'div.woocommerce-profile-wizard__step-header > h2'
 		);
-		expect(pageHeading).toContain('Choose a theme');
+		expect( pageHeading ).toContain( 'Choose a theme' );
 		// Just continue with the current theme
-		await page.click('button >> text=Continue with my active theme');
-	});
-});
+		await page.click( 'button >> text=Continue with my active theme' );
+	} );
+} );
 
 // !Changed from Japanese to Malta store, as Japanese Yen does not use decimals
 test.describe(
 	'A Malta store can complete the selective bundle install but does not include WCPay.',
 	() => {
-		test.use({ storageState: process.env.ADMINSTATE });
+		test.use( { storageState: process.env.ADMINSTATE } );
 
-		test.beforeEach(async ({ page }) => {
-			await onboarding.completeStoreDetailsSection(
-				page,
-				storeDetails.malta.store
-			);
-		});
+		test.beforeEach( async () => {
+			// Complete "Store Details" step through the API to prevent flakiness when run on external sites.
+			await api.update.storeDetails( storeDetails.malta.store );
+		} );
```

Comment:
Same reasoning in https://github.com/woocommerce/woocommerce/pull/35492/files#r1013824706, where I think we need to start the tests with `Malta` pre-selected as the store country to reduce flakiness in cases where they incorrectly start with a US store.